sdk/RELEASE.md

```diff
@@ -1,7 +1,7 @@
 # Current Version (Still in Development)
```

Comment:
Copy the template on top of this and leave it blank, then change this to `2.0.0-alpha.4`.

Reply:
Done.